includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -274,9 +287,20 @@ public static function logs( $data = null ) {
 				'date_modified' => $data->post_modified_gmt,
 			);
 
-			edd_add_api_request_log( $log_data );
+			$meta_to_remove    = array(
+				'_edd_log_request_ip',
+				'_edd_log_user',
+				'_edd_log_key',
+				'_edd_log_token',
+				'_edd_log_version',
+				'_edd_log_time',
+			);
+			$meta_to_migrate   = $post_meta;
+			$new_log_id        = edd_add_api_request_log( $log_data );
+			$add_meta_function = 'edd_add_api_request_log_meta';
 		} else {
-			$post = \WP_Post::get_instance( $data->ID );
+			$post      = \WP_Post::get_instance( $data->ID );
```

Comment:
I know this isn't new code, but I'm questioning whether this is actually necessary? I thought `$data` already had the full object and `post_parent` could be accessed via `$data->post_parent` :thinking:

Reply:
Checked and confirmed, `$data` does look like a post object to me as well.